tests/test_bgp_af.rb

```diff
@@ -745,6 +544,7 @@ def default_metric(asn, vrf, af)
   def test_default_metric
     afs = [%w(ipv4 unicast), %w(ipv6 unicast)]
     afs.each do |af|
+      config_ios_xr_dependencies('55') if platform == :ios_xr
```

Comment:
Seems a little odd to pass `'55` instead of `55` - is there a problem with the function definition?